scio-elasticsearch2/src/test/scala/com/spotify/scio/elasticsearch/ElasticsearchTest.scala

```diff
@@ -44,29 +44,11 @@ object ElasticsearchJob {
   }
 }
 
-object ElasticsearchDirectRunnerJob {
```

Comment:
Why do these ES jobs need to be removed?

Reply:
They look almost identical to the one without `DirectRunner` and I don't think they have anything to do with `DirectRunner` or not. It was just lousy review that they got in the first time ;)